servant-client/src/Servant/Client/Internal/HttpClient.hs

```diff
@@ -68,16 +68,23 @@ import qualified Servant.Types.SourceT                    as S
 import qualified Network.HTTP.Client         as Client
 
 -- | The environment in which a request is run.
+--   The baseUrl and makeClientRequest function are used to create a `http-client` request.
```

Comment:
Note, haddocks aren't markdown. 

```
The 'baseUrl' and 'makeClientRequest' function are used to create a @http-clietn@ request.

...

on a @servant@

```

In short: use ticks, no backticks. `@` for "code-like".

Reply:
Thanks, I fixed the comments in a fixup commit.